book/getting-started/run-a-wallaroo-application-docker.md

```diff
@@ -0,0 +1,154 @@
+# Run a Wallaroo Application in Docker
+
+In this section, we're going to run an example Wallaroo application in Docker. By the time you are finished, you'll have validated that your Docker environment is set up and working correctly.
+
+There's a couple Wallaroo support applications that you'll be interacting with for the first time:
+
+- Our Metrics UI that allows you to monitor the performance and health of your applications.
+- Giles receiver is designed to capture TCP output from Wallaroo applications.
+- Giles sender is used to send test data into Wallaroo applications over TCP.
+- Machida, our program for running Wallaroo Python applications.
+
+You're going to set up our "Celsius to Fahrenheit" example application. Giles sender will be used to pump data into the application. Giles receiver will receive the output and our Metrics UI will be running so you can observe the overall performance.
+
+The Metrics UI process will be run in the background. The other three processes (receiver, sender, and Wallaroo) will run in the foreground. We recommend that you run each process in a separate terminal.
+
+NOTE: You might need to run Docker commands with sudo depending on how you set up Docker.
+
+Let's get started!
+
+## Terminal 1, Start the Wallaroo Docker container
+
+```bash
+docker run --rm -it --publish 4000:4000 --name wally wallaroo-labs-docker-wallaroolabs.bintray.io/dev/wallaroo:0.2.2-42-g4d3277f
+```
+
+We'll be naming our Docker container `wally` so we can access it with convenience and we'll be exposing port 4000 so we can view the Metrics UI. This shell must remain running in order for the container to be kept alive.
+
+Note: For each additional shell needed, run the following command:
```

Comment:
It seems likely that people will not connect your reference of `additional shell` with the `Terminal N` later on and will forget to run the `docker exec` command. It might be better to to be explicit for each new `Terminal N` required and have them run the `docker exec` as the first command in each.

Reply:
@dipinhora this has been addressed